homeassistant/components/deconz/sensor.py

```diff
@@ -139,6 +139,9 @@ def device_state_attributes(self):
         if self._device.on is not None:
             attr[ATTR_ON] = self._device.on
 
+        if self._device.temperature is not None:
+            attr[ATTR_TEMPERATURE] = self._device.temperature
```

Comment:
This one is only available when the device reports temperature in raw["state"].get("temperature"), which is only the case when the device type is ZHATYPE = ("ZHATemperature", "CLIPTemperature").
As you post in your json data the secondary temperature is reported only in raw["config"].get("temperature").

You can look at the pydeconz code here https://github.com/Kane610/deconz/blob/2a51655128148fe77a838097234877f6df6d2b2a/pydeconz/sensor.py#L66